terralab/utils.py

```diff
@@ -29,9 +32,11 @@ def wrapper(*args, **kwargs):
         except ApiException as e:
             formatted_message = f"API call failed with status code {e.status} ({e.reason}): {json.loads(e.body)['message']}"
             LOGGER.error(add_blankline_after(formatted_message))
+            LOGGER.error(add_blankline_after(SUPPORT_EMAIL_TEXT))
             exit(1)
         except Exception as e:
             LOGGER.error(add_blankline_after(str(e)))
+            LOGGER.error(add_blankline_after(SUPPORT_EMAIL_TEXT))
```

Comment:
can we add to the existing tests a check for these in the logs?

Reply:
good idea, yeah